game_patch/multi/server.cpp

```diff
@@ -383,7 +384,13 @@ FunHook<float(rf::Entity*, float, int, int, int)> entity_damage_hook{
             auto* damaged_player_stats = static_cast<PlayerStatsNew*>(damaged_player->stats);
             damaged_player_stats->add_damage_received(real_damage);
 
-            if (g_additional_server_config.hit_sounds.enabled) {
+            auto& pdata = get_player_additional_data(killer_player);
+            if (pdata.multi_hitsounds) { 
+                 if (pdata.multi_hitsounds == 2) {
+                    custom_packets::send_hitsound_packet(killer_player);
+                }
+            }
+            else if (g_additional_server_config.hit_sounds.enabled) {
```

Comment:
I want more configuration for the server owners so they can force hitsounds disabled, no matter what the client configured.
3 server-side modes: enable, disable, force disabled (naming suggestion, can be changed)
3 client-side modes: auto (use server setting), enable (if not forced disabled on the server), disable